go/pkg/sysdb/coordinator/model/collection.go

```diff
@@ -17,6 +17,8 @@ type Collection struct {
 	Version                    int32
 	UpdatedAt                  types.Timestamp
 	TotalRecordsPostCompaction uint64
+	SizeBytesPostCompaction    uint64
```

Comment:
I think we need to make it clear that this is the size of the input log data, not the overall size. A comment would help.

Reply:
This is supposed to represent the total size of the collection? So we can tail updates to it. Unless you were thinking that we only want to track the latest size delta?

Reply:
The total size of the collection data != to the size of the indexes and total actual on s3 storage. You need this to be just the size of the log data for your use case.